src/main/java/com/google/api/gax/grpc/ApiException.java

```diff
@@ -31,17 +31,24 @@
 
 package com.google.api.gax.grpc;
 
+import com.google.common.base.Preconditions;
+
+import io.grpc.Status;
+
 /**
  * Represents an exception thrown during an RPC call.
  *
  * It stores information useful for functionalities in {@link ApiCallable}.
  */
 public class ApiException extends RuntimeException {
   private final boolean retryable;
+  private final Status.Code statusCode;
```

Comment:
Since statusCode comes before retryable in the constructor parameters, put the corresponding members in the same order.